.github/workflows/ansible-test.yml

```diff
@@ -26,6 +29,15 @@ jobs:
 
       - name: Run sanity tests
         run: ansible-test sanity --docker -v --color
+        working-directory: ./ansible_collections/community/grafana
```

Comment:
Any particular reason to add working-directory ? Is it because of a behaviour change in actions for v2 ?

Reply:
Yes, this is because of a behavior change in checkout@v2.

checkout@v2 no longer change the working directory / GITHUB_WORKSPACE to the git project folder after checkout:
|  | workdir before checkout | workdir after checkout |
| --- | --- | --- |
| checkout@v1 | `/home/runner/work/community.grafana/community.grafana` | `/home/runner/work/community.grafana/ansible_collections/community/grafana` |
| checkout@v2 | `/home/runner/work/community.grafana/community.grafana` | `/home/runner/work/community.grafana/community.grafana` |
 
So we have to set the `working-directory` ourselves for `ansible-test` commands. Otherwise the job fails, e.g. without setting `working-directory`:
```
Run ansible-test sanity --docker -v --color --coverage
ERROR: The current working directory must be at or below:

 - an Ansible collection: {...}/ansible_collections/{namespace}/{collection}/

Current working directory: /home/runner/work/community.grafana/community.grafana
``` 

That is how it is done in [ansible-collections/collection_template](https://github.com/ansible-collections/collection_template/blob/0056b1c987a2c519acc59a6d3b91545220c6471a/.github/workflows/ansible-test.yml#L204), [ansible-collections/community.mysql](https://github.com/ansible-collections/community.mysql/blob/c273ee36a7db58a2298cd95a762dbce93a4d03fe/.github/workflows/ansible-test-plugins.yml#L166) and others.